__init__.py

```diff
@@ -84,7 +84,11 @@ def find_and_query(query):
 class StockSkill(MycroftSkill):
     def __init__(self):
         super().__init__()
-        raise Exception('Skill has been disabled by Mycroft')
```

Comment:
ah; I understand now; I thought we were using a generic "disable skill" function;  the changes here indeed accomplish the goal specific to this disablement and explain it more precisely

the only feedback I might have here is that it would be good to have a generic `def set_skill_disabled()` , something like

(needs to live somewhere common to all skills)
``` python
def set_skill_disabled(skill_name, reason, reference_link)
    self.log.info(f"The {skill_name} has been disabled due to {reason}. For more information see {reference_link}")
    raise SkillDisabledException("Skill has been intentionally disabled by Mycroft team")
```

then in the future, (and now) anytime a skill needs to be disabled we can simply call that, and the messaging will be enforced & clear

thoughts?

Reply:
We could. If we were going to do that, it would be good to include it in the MycroftSkill class in mycroft-core. I really don't want this to be a common occurrence though.

We should really consider removing a Skill from the Marketplace if it's going to be in a "deactivated" state for any length of time (like this one has become).

Sounds like this PR addresses the original issue though, so will merge it